SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -229,7 +229,7 @@ bool ASpatialFunctionalTest::IsReady_Implementation()
 
 	for (ASpatialFunctionalTestFlowController* FlowController : FlowControllers)
 	{
-		if (FlowController->IsReadyToRunTest()) // Check if the owner already finished initialization
+		if (FlowController != nullptr && FlowController->IsReadyToRunTest()) // Check if the owner already finished initialization
```

Comment:
why do we need the `!= nullptr` when we remove FlowControllers from the array?

Reply:
Removed the nullptr check